go-selinux/selinux_linux.go

```diff
@@ -282,6 +282,23 @@ func readCon(fpath string) (string, error) {
 	return strings.Trim(retval, "\x00"), nil
 }
 
+func ClassIndex(class string) int {
+	permpath := fmt.Sprintf("class/%s/index", class)
+	indexpath := filepath.Join(getSelinuxMountPoint(), permpath)
+
+	in, err := os.Open(indexpath)
```

Comment:
Not a strong preference, but maybe `ioutil.ReadFile()` and `strconv.Atoi()` will look better?